src/app/work-item/work-item-list/work-item-list.component.scss

```diff
@@ -1,6 +1,54 @@
 @import "../../../assets/stylesheets/color-variables";
 @import "../../../assets/stylesheets/custom";
 
+.add-detailed-wi{
+  text-align: center;
+  padding-top: em(8);
```

Comment:
instead of padding-top we need to align it vertically and horizontally center. use this css for span

position:absolute; top:50%; left: 50%; @include(transform: translate(-50%,-50%));